### PR TITLE
docs: remove auro-radio from index page and rebuild demo js files #354

### PR DESCRIPTION
**Pull request by jason-capsule42**
# Alaska Airlines Pull Request

## Type of change:

Please delete options that are not relevant.

- [ ] New capability
- [ ] Revision of an existing capability
- [ ] Infrastructure change (automation, etc.)
- [x] Other - Docs

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

## Summary by Sourcery

Documentation:
- Remove auro-radio script from the index page of the documentation.

### CODE REVIEW

File: demo/index.min.js [hunk 1]
```diff
@@ -2650,7 +2650,7 @@ class BaseInput extends r {
    */
   handleClickClear() {
     this.inputElement.value = "";
-    this.value = undefined;
+    this.value = "";
```
Comment (sun-mota) [inline demo/index.min.js:2653]: I think it's from a change for another work or there should be a change on `base-input.js` for this.

File: demo/api.min.js [hunk 2]
```diff
@@ -2704,7 +2704,7 @@ class BaseInput extends r {
    */
   handleClickClear() {
     this.inputElement.value = "";
-    this.value = undefined;
+    this.value = "";
```
Comment (sun-mota) [inline demo/api.min.js:2707]: I think it's from a change for another work or there should be a change on `base-input.js` for this.

**Review by sun-mota — CHANGES_REQUESTED** [review]

File: demo/index.min.js (hunk 1 above)
Comment (jason-capsule42) [reply demo/index.min.js:2653]: This is compiled code. The fact that this is changed means a "build" was not done as part of an earlier PR.

**Review by sun-mota — APPROVED** [review]

**Pull request merged**